scripts/lib/hash_reqs.py

```diff
@@ -34,11 +25,17 @@ def expand_reqs(fpath: str) -> List[str]:
     `fpath` can be either an absolute path or a relative path.
     """
     absfpath = os.path.abspath(fpath)
-    output = expand_reqs_helper(absfpath, set())
+    output = expand_reqs_helper(absfpath)
     return sorted(set(output))
 
+def python_version() -> str:
+    """
+    Returns the Python version as string 'major.minor.patchlevel'
+    """
+    return platform.python_version()
```

Comment:
The point of #12868 is to hash the Python that would be used to build the virtualenv, not the currently running Python. If the system Python gets a security upgrade, we want the hash to change even if everything else stays the same, so that we’ll build a new virtualenv with the upgraded Python. We also need to be careful to avoid using `virtualenv-clone` in that case.

Reply:
oh! Thanks.